autogluon/utils/try_import.py

```diff
@@ -82,3 +82,13 @@ def try_import_faiss():
         raise ImportError(
             "Unable to import dependency faiss"
             "A quick tip is to install via `pip install faiss-cpu`. ")
+
+def try_import_fastai_v1():
+    try:
+        from pkg_resources import parse_version  # pylint: disable=import-outside-toplevel
+        import fastai
+        fastai_version = parse_version(fastai.__version__)
+        assert parse_version('1.0.61') <= fastai_version < parse_version('2.0.0'), 'Currently, we only support 1.0.61<=fastai<2.0.0'
+    except OSError as e:
+        raise ImportError("Import fastai failed. A quick tip is to install via `pip install fastai==1.*`. "
+                          "If you are using Mac OSX, please use this torch version to avoid compatibility issues: `pip install torch==1.2.0`.")
```

Comment:
FYI it worked for me on Mac with torch v1.6.0

Reply:
updated to torch 1.6.0